### PR TITLE
more accurate description of `<teiCorpus>`

### PR DESCRIPTION
**Pull request by sydb**
First crack at improving prose of CC for #2445

### CODE REVIEW

**Comment by trishaoconnor** [comment]
Reviewing the prose changes now. It is reading very well so far. Only thing that seems concerning is that all the GitHub checks/test seem to have failed.

File: P5/Source/Guidelines/en/CC-LanguageCorpora.xml [hunk 1]
```diff
@@ -1077,53 +1080,48 @@ determined by an analysis of linguistic features of the text, excluding
...
-indicate the fact, as further discussed in section <ptr target="#CCAS"/>.
- </p>
+indicate the fact, as further discussed in section <ptr target="#CCAS"/>.</p>
 <p>An extended example of one form of linguistic analysis commonly
-practised in corpus linguistics is given in section <ptr target="#AILA"/>.
-</p>
+practised in corpus linguistics is given in section <ptr target="#AILA"/>.</p>
 </div></div>
 <div type="div2" xml:id="CCREC"><head>Recommendations for the Encoding of Large Corpora</head>
 <p>These Guidelines include proposals for the identification and
```
Comment (trishaoconnor) [inline P5/Source/Guidelines/en/CC-LanguageCorpora.xml:1113]: Suggestion for rewording line 1118 / 1117 "behind this catholic approach", perhaps rephrase using 'universal' or 'inclusive' instead?

**Review by trishaoconnor — APPROVED** [review]
The updated prose looks good to me. The examples are clearly explained and address the points raised in the related ticket #2445. I'm happy to approve a merge into dev. The comment above to reword line 1118/1117 is more of a suggestion.

File: P5/Source/Guidelines/en/CC-LanguageCorpora.xml (hunk 1 above)
Comment (raffazizzi) [reply P5/Source/Guidelines/en/CC-LanguageCorpora.xml:1113]: Seconded, and I'd suggest "universal"

Comment (sydb) [reply P5/Source/Guidelines/en/CC-LanguageCorpora.xml:1113]: Changed to “universal” in 7f52ad0fa.

**Pull request merged**